docs/developer.md

```diff
@@ -304,6 +304,10 @@ The `IProtection` parent class provides a helper method for this in `calculate_l
 
 Most exchanges supported by CCXT should work out of the box.
 
+If you need to implement a specific exchange class, these are found in the `freqtrade/exchange` source folder.
```

Comment:
Well adding a new file won't suffice. Freqtrade only finds a new file if it's imported in the `__init__.py` file.

We intentionally don't iterate through submodules.

Reply:
Oh, absolutely. I intentionally didn't got into more detail because it's a Pandora's Box. This was a response simply to address the comment that it wasn't clear where the exchange files lived in the source tree.

Reply:
Sure, but this whole section is marked as work in progress so .... 😆